scripts/rollup.config.js

```diff
@@ -13,11 +13,40 @@ const options = {
   primedPlugins(defaults) {
     defaults.string = string({include: ['test/manifests/*.mpd']});
 
+    return defaults;
+  },
+  externals(defaults) {
+    defaults.module.push('xmldom');
```

Comment:
externals for the `module` output are modules that should be `required` rather than built into the bundle.